docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -11,50 +11,52 @@
-| SimMIM (ViT-B32) |        256 |    800 |              0.351 |  302.8 Min |     10.5 GByte |
-| SimSiam          |        256 |    800 |              0.871 |  178.2 Min |      3.9 GByte |
-| SwaV             |        256 |    800 |              0.899 |  309.0 Min |      6.4 GByte |
-| SwaVQueue        |        256 |    800 |              0.898 |  300.3 Min |      6.4 GByte |
-| SMoG             |        256 |    800 |              0.782 |  250.2 Min |      2.5 GByte |
-| TiCo             |        256 |    800 |              0.857 |  184.7 Min |      2.5 GByte |
-| VICReg           |        256 |    800 |              0.843 |  192.9 Min |      5.7 GByte |
-| VICRegL          |        256 |    800 |              0.781 |  207.4 Min |      5.7 GByte |
----------------------------------------------------------------------------------------------
+Results (4.5.2023):
+-------------------------------------------------------------------------------------------------
+| Model            | Batch Size | Epochs |  KNN Top1 Val Accuracy |       Time | Peak GPU Usage |
+-------------------------------------------------------------------------------------------------
+| BarlowTwins      |        256 |    200 |                  0.651 |   85.0 Min |      4.0 GByte |
+| BYOL             |        256 |    200 |                  0.705 |   54.4 Min |      4.3 GByte |
+| DCL              |        256 |    200 |                  0.809 |   48.7 Min |      3.7 GByte |
+| DCLW             |        256 |    200 |                  0.783 |   47.3 Min |      3.7 GByte |
+| DINO (Res18)     |        256 |    200 |                  0.873 |   75.4 Min |      6.6 GByte |
+| FastSiam         |        256 |    200 |                  0.779 |   88.2 Min |      7.3 GByte |
+| MAE (ViT-S)      |        256 |    200 |                  0.454 |   62.0 Min |      4.4 GByte |
+| MSN (ViT-S)      |        256 |    200 |                  0.713 |  127.0 Min |     14.7 GByte |
+| Moco             |        256 |    200 |                  0.786 |   57.5 Min |      4.3 GByte |
+| NNCLR            |        256 |    200 |                  0.809 |   51.5 Min |      3.8 GByte |
+| PMSN (ViT-S)     |        256 |    200 |                  0.705 |  126.9 Min |     14.7 GByte |
+| SimCLR           |        256 |    200 |                  0.835 |   49.7 Min |      3.7 GByte |
+| SimMIM (ViT-B32) |        256 |    200 |                  0.315 |  115.5 Min |      9.7 GByte |
+| SimSiam          |        256 |    200 |                  0.752 |   58.2 Min |      3.9 GByte |
+| SwaV             |        256 |    200 |                  0.861 |   73.3 Min |      6.4 GByte |
+| SwaVQueue        |        256 |    200 |                  0.827 |   72.6 Min |      6.4 GByte |
+| SMoG             |        256 |    200 |                  0.663 |   58.7 Min |      2.6 GByte |
+| TiCo             |        256 |    200 |                  0.742 |   45.6 Min |      2.5 GByte |
```

Comment:
TiCo also improved a lot. Do you know why?

Reply:
Same as above :)